scripts/publish-vscode-extension.ts

```diff
@@ -19,9 +20,13 @@ async function updateExtensionVersion() {
 	if (!version) {
 		throw new Error('Could not get the version of the published extension.')
 	}
-	const versionSplit = version.split('.')
-	versionSplit[2] = Number(versionSplit[2]) + 1
-	const nextVersion = versionSplit.join('.')
+	const semVer = parse(version)
```

Comment:
did you check that this version number returned by vsce would always be the latest _including_ prerelease versions?

Reply:
Yes I did, that's one of the reasons I published a prerelease. Running `yarn get-info` returns `2.0.37` right now, which is the prerelease.